### PR TITLE
Changed some download links to https

### PR DESCRIPTION
**Pull request by danielmewes**
download.rethinkdb.com now supports https. I switched some of the links over (most importantly the ones where we download a gpg key for the debian/ubuntu packages).

### CODE REVIEW

File: 0-getting-started/install/centos.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ package. Installing these dependencies from the EPEL repositories will
 allow RethinkDB to build more quickly:
 
 ```bash
-sudo rpm -Uvh http://download.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
```
Comment (danielmewes) [inline 0-getting-started/install/centos.md]: The download.fedoraproject.org server also supports SSL. Probably a good idea to use it.

Comment (chrisvariety) [inline 0-getting-started/install/centos.md]: Looks like it redirects to a random non-https mirror, so kind of misleading, e.g.

```
$ curl -IL https://download.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
HTTP/1.1 302 Found
Date: Fri, 28 Aug 2015 01:16:52 GMT
Server: Apache/2.4.6 (Red Hat Enterprise Linux) OpenSSL/1.0.1e-fips mod_wsgi/3.4 Python/2.7.5
Location: http://mirror.sfo12.us.leaseweb.net/epel/6/x86_64/epel-release-6-8.noarch.rpm
AppTime: D=473418
AppServer: proxy04.fedoraproject.org
Content-Type: text/html; charset=UTF-8
```

Comment (danielmewes) [inline 0-getting-started/install/centos.md]: Ah I didn't realize that. Good catch. I'm going to revert that link back to http for now.

**Comment by danielmewes** [comment]
Ping @chipotle. Could you review this?

**Pull request merged**